resource_sharing/utilities.py

```diff
@@ -69,20 +77,70 @@ def repositories_cache_path():
         'repositories_cache')
 
 
+def local_collection_root_dir_key():
+    """The QSettings key for the local collections root dir."""
+    return 'localCollectionDir'
+
+
+def default_local_collection_root_dir():
+    return os.path.join(QgsApplication.qgisSettingsDirPath(),
+                        'resource_sharing',
+                        'collections')
+
+
 def local_collection_path(id=None):
     """Get the path to the local collection dir.
 
     If id is not passed, it will just return the root dir of the collections.
     """
-    # path = Path(QDir.homePath()) / 'QGIS' / 'Resource Sharing')
+    settings = QSettings()
```

Comment:
QgsSettings is always recommended instead of QSettings

Reply:
Thanks!
QSettings is used elsewhere in the plugin code, so I assumed that was the way to do it
(and I was not even aware of the existence of QgsSettings).
What about letting this through here and creating a separate issue on the use of QgsSettings that covers all the code?

Reply:
Works for me. 
QgsSettings was probably introduced after the initial development of this plugin.